Izzy the Iguana, master of Pipes

swifterjack opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds an iguana. Sprites by Torimaesu.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
![iguana_forward](https://user-images.githubusercontent.com/25721803/151852858-0a13371e-4cf5-44b8-a884-87fd7fd51b01.png)

## Why It's Good For The Game
Dude hell yeah iguana pet for the gec!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
iganga
## Changelog
:cl:
add: Iguana
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

swifterjack committed:
iguana

Apogee-dev commented:
don't forget to make iguana whip it good instead of bite

swifterjack committed:
they can now whip, but not naenae

swifterjack commented:
> don't forget to make iguana whip it good instead of bite

they can now whip. just needs a dead sprite, and this should be good

swifterjack committed:
iguana is fucking dead now, nice job asshole

swifterjack committed:
Merge branch 'master-2' into iguanamaxing

swifterjack commented:
this pr is finished, should be good to merge

swifterjack commented:
how do i undraft it

rye-rice commented:
click the ready for review button

Ms-Mee commented:
> how do i undraft it

How to charge batong?

swifterjack commented:
> > how do i undraft it
> 
> How to charge batong?

how do i report you to god for your miscreant deeds

Ms-Mee commented:
> > > how do i undraft it
> > 
> > How to charge batong?
> 
> how do i report you to god for your miscreant deeds

I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine

swifterjack commented:
> > > > how do i undraft it
> > > 
> > > 
> > > How to charge batong?
> > 
> > 
> > how do i report you to god for your miscreant deeds
> 
> I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine

how do i report you to your girlfriend for your miscreant deeds

Ms-Mee commented:
> > > > > how do i undraft it
> > > > 
> > > > 
> > > > How to charge batong?
> > > 
> > > 
> > > how do i report you to god for your miscreant deeds
> > 
> > I answer only to the primal slime's thoughts and my girlfriend, everything else has no management power over mine
> 
> how do i report you to your girlfriend for your miscreant deeds

I mean if you find out who she is she'd be more worried about some random person talking about her girlfriend than whatever thing I did

ZephyrTFA commented on whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm:
```diff
@@ -24,3 +24,23 @@
 	speed = 8
 	glide_size = 2
 	footstep_type = FOOTSTEP_MOB_CLAW
+//shipping my tests all over the floor
```
```suggestion

```

swifterjack committed:
Update whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm

no more meme comment

Co-authored-by: Zephyr <12817816+ZephyrTFA@users.noreply.github.com>

ZephyrTFA commented on whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm:
```diff
@@ -24,3 +24,22 @@
 	speed = 8
 	glide_size = 2
 	footstep_type = FOOTSTEP_MOB_CLAW
+/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
```
```suggestion

/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
```

swifterjack committed:
Update whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm

readding the space that was removed when shipping my tests all over the floor was ILLEGALIZED

Co-authored-by: Zephyr <12817816+ZephyrTFA@users.noreply.github.com>

ZephyrTFA reviewed (approved):
iguana

swifterjack commented:
> iguana

iguana

rye-rice commented:
Computer, activate iguana.

TheNeoGamer42 commented:
> Computer, activate iguana.

Computer, Deactivate iguana inhibitors, MAXIMUM IGUANA, NOW.

MarkSuckerberg commented on whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm:
```diff
@@ -24,3 +24,23 @@
+
+/mob/living/simple_animal/hostile/retaliate/gator/steppy/iguana
+	name = "Izzy"
+	desc = "A master of looking at pipes, she's a favorite among the GEC for her GECK-like nature."
```
```suggestion
	desc = "A master of looking at pipes, she's a favorite among the GEC for her GECK-like nature."
	gender = FEMALE
```
let's keep it consistent

swifterjack committed:
Update whitesands/code/modules/mob/living/simple_animal/friendly/gator.dm

mark

Co-authored-by: Mark Suckerberg <29362068+MarkSuckerberg@users.noreply.github.com>
